api/src/routes/stardust/nftDetails/get.ts

```diff
@@ -16,7 +17,7 @@ import { ValidationHelper } from "../../../utils/validationHelper";
 export async function get(
     config: IConfiguration,
     request: INftDetailsRequest
-): Promise<INftOutputsResponse> {
+): Promise<INftDetailsResponse | unknown> {
```

Comment:
Let's make all field in INfdDetailsResponse optional `?`, so then we can remove this `unknown`.

Reply:
Done